lib/optimizely/bucketer.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
Seems unnecessary.

Reply:
Is it to make it Ruby 2 compatible?

Reply:
Can you see if this comment can be at the end of license text?

Reply:
Looks like it has to be first line: https://stackoverflow.com/questions/37799296/what-does-the-comment-frozen-string-literal-true-do